api/internal/core/entity/entity.go

```diff
@@ -133,16 +134,16 @@ type UnHealthy struct {
 }
 
 type Active struct {
-	Type                   string    `json:"type,omitempty"`
-	Timeout                int       `json:"timeout,omitempty"`
-	Concurrency            int       `json:"concurrency,omitempty"`
-	Host                   string    `json:"host,omitempty"`
-	Port                   int       `json:"port,omitempty"`
-	HTTPPath               string    `json:"http_path,omitempty"`
-	HTTPSVerifyCertificate string    `json:"https_verify_certificate,omitempty"`
-	Healthy                Healthy   `json:"healthy,omitempty"`
-	UnHealthy              UnHealthy `json:"unhealthy,omitempty"`
-	ReqHeaders             []string  `json:"req_headers,omitempty"`
+	Type                   string       `json:"type,omitempty"`
+	Timeout                TimeoutValue `json:"timeout,omitempty"`
```

Comment:
Why change this to float32?

Reply:
There are two "number" definitions in jsonschema, one is an integer, the other is number, where number runs to store float values. Such as 5.5

Therefore, a float value is used here to store.

In addition, only map[string]interface{} was used to store the value before, and there was ambiguity in the data type. Crash may occur during runtime. This is also the reason for this modification.（add a Timeout struct）